evasquez26/angular4-startup
# S1 one 88 left navigation panel

## Story / Task

see [ONE-88](https://bixlabs.atlassian.net/browse/ONE-88)

## What this PR does?

Adds a side menu on the left without much style

### Before 

- We had not left menu

### After

- Added a left navigation panel without much style
- Added authentication service, this will allow in the future to determine if a users is currently logged in or not, that way we can hide or show menu items

### How to test / reproduce

- Open up the app
- Observe that there's a left navigation panel now

### TODOs
- [x] Rebase after merging https://github.com/evasquez26/angular4-startup/pull/2

### Related PRs

N/A

## Migrations

NO
